front/src/index.css

```diff
@@ -263,3 +263,12 @@ div.row {
 .search-bar {
   width: 50px;
 }
+
+.a.rs-dropdown-item-content{
+  padding: 0 0 0 0;
+}
+
+
+.rs-sidenav-collapse-in .rs-dropdown-item>.rs-dropdown-item-content {
+     padding: 10px!important;
+}
```

Comment:
This is the only thing that bothers me - the classes are very strange, cant we just create our own classes and add them to the specific components? It will be more verbose and better for later debug/refactor.

Reply:
> This is the only thing that bothers me - the classes are very strange, cant we just create our own classes and add them to the specific components? It will be more verbose and better for later debug/refactor.

its hard to override this, for now this is only way